tests/test_add_function.py

```diff
@@ -1,68 +1,75 @@
 """Test Triplestore.add_function()"""
 # pylint: disable=invalid-name
-from tripper import Triplestore
 
 
-def func(a, b):
-    """Returns the sum of `a` and `b`."""
-    return a + b
+def test_add_function():
+    """Test add_function()"""
+    import pytest
 
+    pytest.importorskip("rdflib")
+    from tripper import Triplestore
+    from tripper.utils import function_id
 
-ts = Triplestore(backend="rdflib")
-EX = ts.bind("ex", "http://example.com/ex#")
-ts.add_function(
-    func,
-    expects=[EX.arg1, EX.arg2],
-    returns=EX.sum,
-    standard="fno",
-)
-assert (
-    ts.serialize().strip()
-    == """
+    def func(a, b):
+        """Returns the sum of `a` and `b`."""
+        return a + b
+
+    f_id = function_id(func)
```

Comment:
Very nice idea to store the f_id such that it can be reused in the assertments below :-)

Reply:
Thanks :) That was my idea